webapp/components/test-file-results-table.js

```diff
@@ -412,6 +419,28 @@ class TestFileResultsTable extends WPTFlags(Pluralizer(AmendMetadataMixin(WPTCol
   openAmendMetadata() {
     this.$.amend.open();
   }
+
+  shouldDisplayMetadata(index, subtestname, metadataMap) {
+    if (!metadataMap) {
+      return false;
+    }
+    return this.displayMetadata && this.getMetadataUrl(index, subtestname, metadataMap) !== '';
+  }
+
+  getMetadataUrl(index, subtestname, metadataMap) {
```

Comment:
Is this copied from `wpt-results.js`? Can we share them?

Reply:
It is a bit different. `metadataMap` is a nested map with a format {`key1`: {`key2`: url}} where `key1` is the test name and `key2` is a subtest name or `/`. So the way it retrieves the URL is different between wpt-result.js and test-file-results-table.js.